source/includes/kubernetes_extension/_k8_secrets.md

```diff
@@ -42,14 +42,14 @@ Retrieve a list of all secrets in a given [environment](#administration-environm
 | -------------------------- | --------------------------------------------------- |
 | `cluster_id` <br/>_string_ | The id of the cluster in which to list the secrets. |
 
-| Attributes                 | &nbsp;                                                                     |
-| -------------------------- | ---------------------------------------------------------------------------|
-| `id` <br/>_string_         | The id of the secret.                                                      |
-| `apiVersion` <br/>_string_ | The API version used to retrieve the secret.                               |
-| `data` <br/>_string_       | The data that the secret contains.                                         |
-| `metadata` <br/>_object_   | The metadata of the secret.                                                |
-| `kind` <br/>_string_       | A string value representing the REST resource that this object represents. |
-| `type` <br/>_string_       | A string used to facilitate programmatic handling of a secret's data.      |
+| Attributes                  | &nbsp;                                                                     |
+| --------------------------- | -------------------------------------------------------------------------- |
+| `id` <br/>_string_          | The id of the secret.                                                      |
+| `apiVersion` <br/>_string_  | The API version used to retrieve the secret.                               |
+| `encodedData` <br/>_string_ | The data that the secret contains.                                         |
```

Comment:
If we decide to make changes as per the previous comment then we should make changes here as well else ignore this comment.

Reply:
a855197d879b83a378f7bb703864deb33ea8ed53